packages/typespec-client-generator-core/test/types/multipart-types.test.ts

```diff
@@ -454,6 +455,52 @@ describe("typespec-client-generator-core: multipart types", () => {
     strictEqual(fileRequiredFileName.multipartOptions.contentType.optional, false);
   });
 
+  it("with MultiPartFile of Azure.Core", async function () {
```

Comment:
we also want tests in `azure-core` for this

Reply:
added